src/utils/navigatorData.js

```diff
@@ -179,16 +179,21 @@ export function getSiblings(uid, childrenMap, children) {
 }
 
 function extractRootNode(data) {
+  // note: this "root" path won't always necessarily come at the beginning of
+  // the URL in situations where the renderer is being hosted at some path
+  // prefix
```

Comment:
Since we know the value of `baseUrl`, is it possible to match the `node.path` at the exactly correct position in the current page’s URL, i.e. directly following `baseUrl`, instead of allowing it to match anywhere?

Reply:
Good idea. I'll explore that.